From 8babc442a5dc21329b19bd4e4c09e2688a0713d9 Mon Sep 17 00:00:00 2001
From: Juan Carlos Niebles <niebles@gmail.com>
Date: Mon, 2 Dec 2024 11:37:19 -0800
Subject: [PATCH 09/20] Fix to display references in order of citation (#2855)

"cite_in_order" here contradicts the global group_by year setting in
_config.yml
---
 _layouts/post.liquid | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_layouts/post.liquid b/_layouts/post.liquid
index 89e1e6ad..5e00c922 100644
--- a/_layouts/post.liquid
+++ b/_layouts/post.liquid
@@ -80,7 +80,7 @@ layout: default
   {% if page.related_publications %}
     <h2>References</h2>
     <div class="publications">
-      {% bibliography --cited_in_order %}
+      {% bibliography --group_by none --cited_in_order %}
     </div>
   {% endif %}
 
-- 
2.48.1

